src/lib/core.js

```diff
@@ -3466,7 +3466,7 @@ export default function (context, pluginCallButtons, plugins, lang, _options) {
             const commandMapNodes = [];
             const currentNodes = [];
 
-            let findFormat = true, findAlign = true, findList = true, findFont = true, findSize = true, findOutdent = true, findA = true;
+            let findFormat = true, findAlign = true, findList = true, findFont = true, findSize = true, findOutdent = true, findA = true, findMath = true;

```

Comment:
I am modifying that function in the "develop" brunch.
You do not need to modify this function.
I'll add :)

Reply:
sure